src/config/media/index.md

```diff
@@ -3,6 +3,8 @@
 ## Audio setup
 
 To setup audio on your Void Linux system you have to decide if you want to use
-[PulseAudio](./pulseaudio.md) or just [ALSA](./alsa.md).
+[PulseAudio](./pulseaudio.md), [PipeWire](./pipewire.md) or just
+[ALSA](./alsa.md).
 
-Some applications require PulseAudio, especially closed source programs.
+Some applications require PulseAudio, especially closed source programs, but
+[PipeWire](./pipewire.md) provides a drop-in PulseAudio replacement.
```

Comment:
`<..> drop-in replacement for PulseAudio.`

Reply:
:+1: